core/src/kmx/kmx_plus.h

```diff
@@ -34,6 +34,8 @@ namespace kmx {
 #pragma warning ( disable : 4200 )
 #endif
 
+typedef KMX_DWORD KMXPLUS_STR;
```

Comment:
Has no real meaning but helps to clarify that these are offsets to strings rather than plain DWORDs 😁